models/prices/optimism/prices_optimism_tokens.sql

```diff
@@ -3,22 +3,19 @@
         alias = alias('tokens'),
         materialized='table',
         file_format = 'delta',
-        tags=['static']
+        tags = ['static', 'dunesql']
         )
 }}
 SELECT
-    token_id
-    , blockchain
-    , symbol
-    , contract_address
-    , decimals
-FROM {{ ref('prices_optimism_tokens_curated') }}
```

Comment:
I think we need to include curated here still

Reply:
per @MSilb7 recommendation in earlier comments on this PR, we can remove these. optimism will rely on curated manual adds from the ethereum file and inner join to his other spell for bridged mappings of addresses to filter down the results optimism cares about.

@MSilb7 can you confirm once again we're good here for optimism?

Reply:
Regarding selecting only from `tokens_optimism_erc20`, we still need the coinpaprika ids. But I agree that it could be just one file. If we do it the way we have now, people can only see prices for tokens that are in ethereum. So if there's a token that exists on optimism but not ethereum it's not going to work, or if the address is different.

Reply:
cc @aalan3 & @jeff-dude , the optimism prices file should stay as it was before. I had the erc20 tokens file and the prices file confused.

Reply:
thank you. i've reverted OP back @aalan3